adal/src/main/java/com/microsoft/aad/adal/ADALError.java

```diff
@@ -344,11 +344,18 @@
      */
     BROKER_PACKAGE_NAME_NOT_FOUND("Broker is not installed in your system"),
 
+    BROKER_PRT_FAILED("Broker is failed to get PRT"),
```

Comment:
Grammatical issue with this error, should be `Broker failed to get PRT`

Reply:
this is moved from Unity. good catch. I will edit this.